libsolidity/interface/CompilerStack.cpp

```diff
@@ -70,11 +70,14 @@
 #include <libsolutil/SwarmHash.h>
 #include <libsolutil/IpfsHash.h>
 #include <libsolutil/JSON.h>
+#include <libsolutil/Algorithms.h>
 
 #include <json/json.h>
 
 #include <boost/algorithm/string/replace.hpp>
 #include <utility>
+#include <map>
+#include <range/v3/view/concat.hpp>
```

Comment:
We usually put `boost` and `ranges` in a group separate from stdlib.

Reply:
I mostly meant boost. Good point about ranges though. Maybe we should start grouping them with stdlib.

Reply:
To my defernce, that boost include was there before I touched that part :P I'll move it down to it's own block.